drift/exceptions.py

```diff
@@ -0,0 +1,14 @@
+class BadRequest(Exception):
+    status_code = 400
```

Comment:
Trying to recall my python vs. ruby. Given this is a constant, should it be STATUS_CODE ?

Reply:
This is some boilerplate from the flask docs on raising exceptions. However it still seems a little strange to me that BadRequest is something we have to define. I'll look into it more today.